compiler/src/iree/compiler/Pipelines/Options.h

```diff
@@ -59,11 +59,11 @@ struct InputDialectOptions {
 // Options controlling high level optimizations.
 struct HighLevelOptimizationOptions {
   // Enables const-expr hoisting into globals.
-  bool constExprHoisting = false;
+  bool constExprHoisting = true;
 
   // Enables recursive evaluation of immutable globals using the compiler
   // and runtime.
-  bool constEval = false;
+  bool constEval = true;
```

Comment:
Can you check that https://iree-org.github.io/iree/reference/optimization-options/ is still up to date after flipping these? Source here: https://github.com/iree-org/iree/blob/main/docs/website/docs/reference/optimization-options.md